src/commands/createWebApp/stacks/getStackPicks.ts

```diff
@@ -24,9 +24,10 @@ export async function getStackPicks(context: IWebAppWizardContext, javaVersion?:
 
             for (const minorVersion of minorVersions) {
                 let description: string | undefined;
-                if ((!javaVersion && (minorVersion.stackSettings.linuxRuntimeSettings?.isPreview || minorVersion.stackSettings.windowsRuntimeSettings?.isPreview)) ||
-                    (javaVersion && (minorVersion.stackSettings.linuxContainerSettings?.isPreview || minorVersion.stackSettings.windowsContainerSettings?.isPreview))) {
+                if (isFlagSet(minorVersion.stackSettings, 'isPreview')) {
```

Comment:
Should we still confirm it's not a Java version, whether that be in this conditional or in the function on line 45?

I recognize that JavaContainers seem to have LinuxJavaContainerSettings and windows settings, but this isn't the behavior that we had previously.

Reply:
Discussed offline